Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs

```diff
@@ -193,11 +192,8 @@ private void Initialize()
                 AddManager(typeof(IMixedRealityTeleportSystem), Activator.CreateInstance(ActiveProfile.TeleportSystemSystemType) as IMixedRealityTeleportSystem);
             }
 
-            #region ActiveSDK Discovery
-
-            // TODO Microsoft.MixedReality.Toolkit - Active SDK Discovery
-
-            #endregion ActiveSDK Discovery
+            // Enable the Voice Manager, ensures voice controllers can be automatically started if the profile is enabled and register voice controllers later if need be.
+            AddManager(typeof(VoiceInputDeviceManager), new VoiceInputDeviceManager("VoiceInput", 10));
```

Comment:
Shouldn't this be moved to where the input manager is added? That way if the input system isn't enabled the voice input device manager is also not added.

Reply:
Scratch that. We already decided long ago that the devices could still be created and we separated them from the input system.